.chloggen/spanner_fgac_user.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Could you explain how this becomes a breaking change?

Reply:
This PR changes the way databases are mentioned in the config file.

Earlier:
```
databases:
            - "db11"
            - "db12"
```

Now:
```
databases:
            - database_name: "db11"
              fgac_database_role: "database_role"
            - database_name: "db12"`
```